src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -34,36 +31,42 @@
         <div class="completed-reqCourses-course-object-semester">in {{ semesterLabel }}</div>
       </div>
     </div>
+    <slot-menu
+      v-model="slotMenuOpen"
+      v-if="slotMenuOpen"
+      :position="position"
+      @open-delete-slot-modal="onReset"
+    />
   </div>
 </template>
 
 <script lang="ts">
 import { PropType, defineComponent } from 'vue';
 import ReqCourse from '@/components/Requirements/ReqCourse.vue';
+import SlotMenu from '@/components/Modals/SlotMenu.vue';
 import DeleteCourseModal from '@/components/Modals/DeleteCourseModal.vue';
 import store from '@/store';
 import { deleteCourseFromSemesters } from '@/global-firestore-data';
 import { onboardingDataCollection } from '@/firebaseConfig';
-import getCurrentSeason, { getCurrentYear } from '@/utilities';
+import getCurrentSeason, { getCurrentYear, clickOutside } from '@/utilities';
 
 const transferCreditColor = 'DA4A4A'; // Arbitrary color for transfer credit
 
 export default defineComponent({
-  components: { ReqCourse, DeleteCourseModal },
+  components: { ReqCourse, DeleteCourseModal, SlotMenu },
   props: {
     slotName: { type: String, required: true },
     courseTaken: { type: Object as PropType<CourseTaken>, required: true },
   },
   data: () => ({
-    resetConfirmVisible: false,
+    deleteModalVisible: false,
+    slotMenuOpen: false,
+    position: { x: 0, y: 0 },
```

Comment:
Consider making position a computed prop so that as the window size changes, the position is recomputed


https://user-images.githubusercontent.com/54298311/118394375-5401c200-b612-11eb-880b-63981769dead.mov